documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -0,0 +1,322 @@
+| ``bool IsAscending``        | Sort order. Ascending: ``true``; Descending: ``false``.  |
++-----------------------------+----------------------------------------------------------+
+| ``std::uint8_t RadixBits``  | Number of bits to sort per a radix sort algorithm pass.  |
+|                             | Only ``8`` is currently supported.                       |
++-----------------------------+----------------------------------------------------------+
+
+
+Parameters
+----------
+
++------------------------------------------------------+------------------------------------------------------------------+
+| Name                                                 | Description                                                      |
++======================================================+==================================================================+
+|  ``q``                                               | SYCL queue to submit the kernel template to.                     |
++------------------------------------------------------+------------------------------------------------------------------+
+|                                                      | The sequences(s) of elements to apply the algorithm to.          |
+|  - ``rng`` (1)                                       | They can be provided as:                                         |
+|  - ``first``, ``last`` (2)                           |                                                                  |
+|  - ``keys_rng``, ``vals_rng`` (3)                    | - ``sycl::buffer`` (1,3),                                        |
+|  - ``keys_first``, ``keys_last``, ``vals_first`` (4) | - ``oneapi::dpl::experimental::ranges::views::all`` (1,3)        |
+|                                                      | - ``oneapi::dpl::experimental::ranges::views::subrange`` (1,3)   |
+|                                                      | - USM pointer (2,4)                                              |
+|                                                      | - ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` (2,4)          |
++------------------------------------------------------+------------------------------------------------------------------+
+|  ``param``                                           | Kernel configuration structure. ``data_per_workitem``,           |
+|                                                      | can be any value among ``32``, ``64``, ``96``,..., ``k * 32``;   |
+|                                                      | ``workgroup_size`` can be only ``64``.                           |
++------------------------------------------------------+------------------------------------------------------------------+
+
+
+Return Value
+------------
+
+``sycl::event`` object representing a status of the algorithm execution.
+
+
+Memory Usage
+------------
+
+.. _local-memory:
+
+Local Memory
```

Comment:
Should this be "Local Memory Requirements"?

Reply:
See previous comment.

Reply:
I've changed it